ansible-role-requirements.yml

```diff
@@ -10,3 +10,10 @@
   scm: git
   src: https://github.com/ceph/ansible-ceph-osd.git
   version: 434a13f4d3ee3f60c410052d2670d5dfea034bf8
+# Created for JIRA RLM-322
+# Frank: this is a forked ansible role for removing
+# hardcoded period mark from hostname template.
+- name: openstack_hosts
+  scm: git
+  src: https://github.com/rcbops/openstack-ansible-openstack_hosts
+  version: ce249ec4386d013d76b617f009a93a408f76619e 
```

Comment:
Merged in PR on openstack-ansible-openstack_hosts, updated stable/newton version is now: 235ec6f0fc5570e7c657b243bcf4208cc2c7739c

Reply:
cool, update is on the way.